go/batch-submitter/flags/flags.go

```diff
@@ -66,6 +66,20 @@ var (
 		Required: true,
 		EnvVar:   prefixEnvVar("MAX_L1_TX_SIZE"),
 	}
+	MinStateRootElementsFlag = cli.Uint64Flag{
+		Name: "min-state-root-elements",
+		Usage: "Minimum number of elements required to submit a state " +
+			"root batch",
+		Required: true,
```

Comment:
We could put a default of 1 here for simplicity

Reply:
Ended up leaving it as optional, as this mirrors the min/max tx size